src/client/pages/OfferNew/utils.ts

```diff
@@ -243,6 +244,27 @@ export const apartmentTypeTextKeys: Record<ApartmentType, string> = {
   [ApartmentType.StudentBrf]: 'CHECKOUT_INSURANCE_APARTMENT_TYPE_BRF',
 }
 
+export const getInsuranceTitle = (
+  offerData: OfferData,
+  textKeys: TextKeyMap,
+) => {
+  if (!isBundle(offerData)) {
+    return textKeys[insuranceTypeTextKeys[offerData.quotes[0].contractType]]()
+  }
+
+  if (isNorwegian(offerData)) {
+    return textKeys.SIDEBAR_INSURANCE_TYPE_NO_BUNDLE()
+  }
+
+  if (isDanish(offerData)) {
+    return offerData.quotes.length === 2
+      ? textKeys.SIDEBAR_INSURANCE_TYPE_DK_CONTENTS_ACCIDENT()
+      : textKeys.SIDEBAR_INSURANCE_TYPE_DK_CONTENTS_ACCIDENT_TRAVEL()
+  }
+
+  return ''
+}
+
```

Comment:
Could be written like this of course but I prefer keeping it clean with no nested ifs 😌
```suggestion
) => {
  if (isBundle(offerData)) {
    if (isNorwegian(offerData)) {
      return textKeys.SIDEBAR_INSURANCE_TYPE_NO_BUNDLE()
    }

    if (isDanish(offerData)) {
      return offerData.quotes.length === 2
        ? textKeys.SIDEBAR_INSURANCE_TYPE_DK_HOME_ACCIDENT()
        : textKeys.SIDEBAR_INSURANCE_TYPE_DK_HOME_ACCIDENT_TRAVEL()
    }
  }

  return textKeys[insuranceTypeTextKeys[offerData.quotes[0].contractType]]()
}
```